kolibri/plugins/learn/assets/src/views/ExploreLibrariesPage/index.vue

```diff
@@ -18,51 +18,56 @@
         {{ $tr('showingLibraries') }}
       </p>
     </div>
-    <LibraryItem
```

Comment:
This is a horrendously confusing diff, but I assume this is just adding an if else?

Reply:
yes